modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/fragments/raylife/fragments/congrats-information/index.css

```diff
@@ -29,6 +29,14 @@
 	position: static;
 }
 
+.congrats-info-price-discount{
+	border-radius: 3px;
```

Comment:
Please, use dialect and variables when is possible.

Reply:
I realized that no element would be reused from the parent component, so I decided to work with a different class. I just changed the class name.

Reply:
Done!